package.json

```diff
@@ -7,7 +7,8 @@
   "author": "Ulf Gebhardt <ulf.gebhardt@webcraft-media.de>",
   "license": "Apache-2.0",
   "scripts": {
-    "release": "scripts/release.sh"
+    "release": "scripts/release.sh",
+    "build": "yarn && cd database && yarn && cd ../frontend && yarn && cd ../admin && yarn && cd ../backend && yarn && cd ../e2e-tests && yarn && cd ../federation && yarn && cd .."
```

Comment:
This is install, not build

Reply:
I do not really see the use case of this change - why and when do we need this?

Reply:
I aggree with such script, because i and also other team-members had several times ago the situation, that some modules were not uptodate. In these cases a manually navigation in each module directory was necessary to start an installation update.

So why should such a helper script not be helpfull or in your @ulfgebhardt words be a usecase?